site/app/templates/grading/electronic/Details.twig

```diff
@@ -604,18 +606,18 @@
 
 {# Total Point Summary (peer) #}
 {% macro render_column_total_peer(context, section, graded_gradeable, index, info, column) %}
-    {# todo: test this (no way to create peer assignments currently) #}
-    {# todo: this is a lot of math for the view, we should move this into the Gradeable model #}
+{#     todo: test this (no way to create peer assignments currently)#}
+{#     todo: this is a lot of math for the view, we should move this into the Gradeable model#}
```

Comment:
are these changes meant to be included ?

Reply:
I was trying to comment something out for testing and I guess I messed it up a bit, I'll remove those changes